.editorconfig

```diff
@@ -20,3 +20,8 @@ trim_trailing_whitespace = false
 [*.js]
 indent_style = tab
 indent_size = 4
+
+[*.yml]
+indent_style = space
```

Comment:
Why not tabs and for JS?

Reply:
It is just a reused .editorconfig and well... this repo does not even contains JS

Reply:
ok